packages/prisma/seed.ts

```diff
@@ -807,7 +820,7 @@ async function main() {
           memberData: {
             email: "owner1-acme@example.com",
             password: "owner1-acme",
-            username: "owner1-acme",
```

Comment:
Keeping the username outside the organization same in the seeder as that is the case for now.